cmd/minishift/cmd/root.go

```diff
@@ -141,6 +140,11 @@ var RootCmd = &cobra.Command{
 		}
 
 		setDefaultActiveProfile()
+
+		// Adding minishift version information to debug logs
+		if glog.V(2) {
+			fmt.Println(fmt.Sprintf("-- minishift version: v%s+%s", version.GetMinishiftVersion(), version.GetCommitSha()))
```

Comment:
```
$ minishift version
minishift v1.11.0+d219f961

$ minishift start -v 3
-- minishift version: v1.11.0+d219f961
-- Starting profile 'minishift'
[...]
```

Though I am still not sure if we should use `Minishift version: v1.11.0+d219f961`. WDYT @praveenkumar ?

Reply:
It's binary version for `Minishift` project so I am fine with the current implementation.

Reply:
@budhrg It's binary version for Minishift project so I am fine with the current implementation.